pom.xml

```diff
@@ -90,7 +90,7 @@
                             <goal>jar</goal>
                         </goals>
                         <configuration>
-                            <failOnError>true</failOnError>
+                            <failOnError>false</failOnError>
```

Comment:
What is the reason for this change?

Reply:
Sorry I forgot the reason for this change (^-^). Keep `true` should be ok. Could you solve this on your side or I have to close this PR and re-commit a new one?

By the way, jgitver extension does not work correctly on my side, which always make 1.1.2-SNAPSHOT version.

Reply:
Let's keep it as is, would be extra cool if you could reflect that in the CHANGEFLOG as a separate bullet point. 

About jgitver: that's weird, probably you're testing it off a fork and there are no tags fetched; that's what the master branch prints me:

```bash
(master)]$ mvn package
[INFO] Using jgitver configuration file: /Users/.../proj4j/.mvn/jgitver.config.xml
[INFO] Using jgitver-maven-plugin [1.8.0] (sha1: 3f3523c5f7540865e3ad66fc9ef1c9daa52a6e6b)
[INFO]     version '1.1.6-SNAPSHOT' computed in 1340 ms
[INFO]
[INFO] Scanning for projects...
[INFO] jgitver-maven-plugin is about to change project(s) version(s)
[INFO]     org.locationtech.proj4j::proj4j::1.1.6-SNAPSHOT -> 1.1.6-SNAPSHOT
[INFO]
[INFO] -------------------< org.locationtech.proj4j:proj4j >-------------------
[INFO] Building Proj4J 1.1.6-SNAPSHOT
```

Reply:
I have revised it as `true` now.